source/backend/api/Models/Concepts/Property/PropertyModel.cs

```diff
@@ -222,6 +222,11 @@ public class PropertyModel : BaseModel
         /// </summary>
         public string GeneralLocation { get; set; }
 
+        /// <summary>
+        /// get/set - Property contacts.
+        /// </summary>
+        public List<PropertyContactModel> PropertyContacts { get; set; }
+
```

Comment:
question - I did not see the corresponding `propertyContacts` array on the frontend API model. Is this intended?

Reply:
This is a minimun set of changes. Once we get the scaffold the models will need to be aligned